libecs/CMakeLists.txt

```diff
@@ -35,65 +35,65 @@ fetch_boost_library(predef)
 fetch_boost_library(io)
 
 fetchcontent_declare(
-    nlohmann_json
-    GIT_REPOSITORY https://github.com/nlohmann/json.git
-    GIT_TAG v3.10.5
+        nlohmann_json
+        GIT_REPOSITORY https://github.com/nlohmann/json.git
+        GIT_TAG v3.10.5
 )
-if(NOT nlohmann_json_POPULATED)
+if (NOT nlohmann_json_POPULATED)
     message(STATUS "Populating json")
     fetchcontent_populate(nlohmann_json)
     add_subdirectory(${nlohmann_json_SOURCE_DIR} ${nlohmann_json_BINARY_DIR})
-endif()
+endif ()
 
 add_library(
-    ${PROJECT_NAME}
-    source/Core/Scene.cxx
-    source/Core/EntityManager.cxx
-    source/Core/SceneManager.cxx
-    source/Core/Component/description.cxx
-    source/Core/Component/index.cxx
-    source/Core/Systems/description.cxx
-    source/Core/Systems/index.cxx
-    source/Core/Systems/manager.cxx
-    source/Core/Component/manager.cxx
-    source/Core/Component/ref.cxx
-    source/Core/Component/combination.cxx
-    source/Core/Component/ScriptingComponentArray.cxx
-    source/Core/Data/value.cxx
-    source/Core/Data/value_serialization.cxx
-    source/Core/Data/type.cxx
-    source/Core/Event/index.cxx
-    source/Core/Event/description.cxx
-    source/Core/Event/manager.cxx
-    source/Components/TagArray.cxx
-    source/Components/Tag.cxx
-    source/Components/RigidBody.cxx
-    source/Components/Gravity.cxx
+        ${PROJECT_NAME}
+        source/Core/Scene.cxx
+        source/Core/EntityManager.cxx
+        source/Core/SceneManager.cxx
+        source/Core/Component/description.cxx
+        source/Core/Component/index.cxx
+        source/Core/Systems/description.cxx
+        source/Core/Systems/index.cxx
+        source/Core/Systems/manager.cxx
+        source/Core/Component/manager.cxx
+        source/Core/Component/ref.cxx
+        source/Core/Component/combination.cxx
+        source/Core/Component/ScriptingComponentArray.cxx
+        source/Core/Data/value.cxx
+        source/Core/Data/value_serialization.cxx
+        source/Core/Data/type.cxx
+        source/Core/Event/index.cxx
+        source/Core/Event/description.cxx
+        source/Core/Event/manager.cxx
+        source/Components/TagArray.cxx
+        source/Components/Tag.cxx
+        source/Components/RigidBody.cxx
+        source/Components/Gravity.cxx
 )
 target_include_directories(${PROJECT_NAME} PUBLIC include/)
```

Comment:
Can you remove the formatting change, please? Use cmake-format

Reply:
Sure